library/Director/Web/Table/ActivityLogTable.php

```diff
@@ -213,6 +213,13 @@ protected function makeLink($row)
             // multi column key :(
             if ($type === 'service' || $this->hasObjectFilter) {
                 $object = "\"$name\"";
+            } elseif ($type === 'scheduled_downtime') {
+                $object = Link::create(
+                    "\"$name\"",
+                    'director/' . str_replace('_', '-', $type),
+                    ['name' => $name],
+                    ['title' => $this->translate('Jump to this object')]
+                );
             } else {
                 $object = Link::create(
                     "\"$name\"",
```

Comment:
I would instead handle this in the `else` statement like below. Otherwise, the solution looks acceptable to me.

```suggestion
            } else {
                $delimiter = $type === 'scheduled_downtime' ? '-' : '';
                $object = Link::create(
                    "\"$name\"",
                    'director/' . str_replace('_', $delimiter, $type),
                    ['name' => $name],
                    ['title' => $this->translate('Jump to this object')]
                );
            }
```

Reply:
@moreamazingnick I tried to force push, but unfortunately I don't have the rights to do so.